.github/workflows/build-smoke-tests.yml

```diff
@@ -59,8 +77,12 @@ jobs:
             folder: vue/quasar-app
             executable: app
         include:
+          - platform: ubuntu-latest
+            ext: _0.1.0_amd64.deb
+          - platform: macos-latest
+            ext: .app
```

Comment:
we need to actually descend a bit deeper for macos - on my mac, they end up e.g. here:

`examples/vue/quasar-app/src-tauri/target/release/bundle/osx/app.app`

Reply:
👍 I added another property that we can specify the release folder. However, is there any way we can make this more standard between platforms? I expect cross-platform building will be a selling point and that consistency would be helpful.

Reply:
This is something to address in the cargo bundler - but yes it should definitely be possible.